website/content/docs/ecosystem.mdx

```diff
@@ -6,194 +6,86 @@ description: Comparison between Nomad and Kubernetes
 
 # Nomad Ecosystem
 
-### Continuous Integration & Delivery
+## Continuous Integration & Delivery
 
-#### GitLab
+- [GitLab](https://www.hashicorp.com/resources/nomad-ci-cd-developer-workflows-and-integrations)
+- [Codefresh](https://codefresh.io/docs/docs/yaml-examples/examples/nomad/)
+- [CircleCI](https://circleci.com/docs/2.0/nomad/)
+- [Drone](https://docs.drone.io/runner/nomad/overview/)
+- [Jenkins](https://plugins.jenkins.io/nomad/)
+- [Buildkite](https://buildkite.com/works-with/hashicorp)
 
-https://www.hashicorp.com/resources/nomad-ci-cd-developer-workflows-and-integrations
+## Container Runtime
 
-#### Codefresh
+- [Containerd](https://www.nomadproject.io/plugins/drivers/community/containerd)
+- [Windows IIS](https://github.com/Roblox/nomad-driver-iis)
+- [Jailtask](https://www.nomadproject.io/plugins/drivers/community/jail-task-driver)
+- [Pot](https://github.com/trivago/nomad-pot-driver)
+- [Singularity](https://www.nomadproject.io/plugins/drivers/community/singularity)
+- [Firecracker](https://www.nomadproject.io/plugins/drivers/community/firecracker-task-driver)
+- [Podman](https://www.nomadproject.io/plugins/drivers/podman)
+- [LXC](https://www.nomadproject.io/plugins/drivers/community/lxc)
 
-https://codefresh.io/docs/docs/yaml-examples/examples/nomad/
+## Application Definition & Image Build
 
-#### CircleCI
+- [Levant](https://github.com/hashicorp/levant)
+- [Packer](https://www.packer.io/)
+- [Waypoint](https://learn.hashicorp.com/collections/waypoint/get-started-nomad)
 
-https://circleci.com/docs/2.0/nomad/
+## Container Registry
 
-#### Drone
+- [JFrog Artifactory](https://jfrog.com/blog/cluster-management-made-simple-with-jfrog-artifactory-and-hashicorp-nomad/)
 
-https://docs.drone.io/runner/nomad/overview/
+## Observability and Analysis
 
-#### Jenkins
+- Prometheus
+  - [Using Prometheus to Monitor Nomad Metrics](https://learn.hashicorp.com/tutorials/nomad/prometheus-metrics)
+  - [Start Prometheus](https://learn.hashicorp.com/tutorials/nomad/dynamic-application-sizing?in=nomad/nomad-1-0#start-prometheus)
+- [Grafana](https://www.metricfire.com/blog/monitoring-hashicorp-nomad-with-prometheus-and-grafana/)
+- [DataDog](https://docs.datadoghq.com/integrations/nomad/)
+- [Turbonomics](https://www.youtube.com/watch?v=lwtIaPpdDsc)
+- [Circonus](https://www.hashicorp.com/integrations/circonus/nomad)
+- [Flowmill](https://www.flowmill.com/blog/announcing-flowmill-s-integration-with-nomad/)
```

Comment:
Dead domain; looks like flowmill got eaten by Splunk and they didn't keep the site:

```suggestion
```